include/circt/Dialect/FIRRTL/FIRRTLOps.h

```diff
@@ -23,6 +23,7 @@
 #include "mlir/IR/SymbolTable.h"
 #include "mlir/Interfaces/InferTypeOpInterface.h"
 #include "mlir/Interfaces/SideEffectInterfaces.h"
+#include "llvm/ADT/APSInt.h"
```

Comment:
Is this actually needed in `FIRRTLOps.h` or is it something that should be included in the files that use it?

Reply:
Good point, have moved this to the cpp files.